content/02kapitel.tex

```diff
@@ -39,5 +39,37 @@ \section{Rollenverteilung}
+    bezüglich des jeweiligen Arbeitsaufwandes eingeschätzt und zum anderen gibt jedes Teammitglied eine Kapazität (in Story-Points) für den Sprint an.
+    Das Resultat eines \glqq Sprint Plannings\grqq~ist das Sprint-Ziel und der Sprint Backlog.
+
+    Um im Voraus Storys für das Sprint Planning zu erstellen und auszuplanen erfolgt das \enquote{Roadmap-Meeting}.
+    Hierbei wurden die Storys, welche vom PO erstellt wurden, mit dem Architekten abgesprochen.
+    Ziel ist es innerhalb dieses Meetings Storys auf ihre technische Umsetzbarkeit zu kontrollieren und möglicherweise in kleinere aufzuteilen.
+    Dieses Meeting fand meistens an den Montagen vor den jeweiligen Estimation Meetings statt.
+
+    Innerhalb der \glqq Estimation-Meetings\grqq~werden neue, meist vom Product Owner erstellte Tickets im Backlog von den jeweiligen Service-Leads
+    vorgestellt, deren Service vom Ticket betroffen ist (oder dem Scrum Master, sollte kein Service direkt betroffen sein). Nachdem Fragen zum Ticket
+    geklärt wurden, wird der Arbeitsaufwand des Tickets in Story-Points geschätzt, wobei Fibonacci-Zahlen von 1 bis 21 verwendet werden. Zum Schätzen
+    gibt jedes Teammitglied eine Zahl an (oder enthält sich) und am Ende wird die Zahl mit den meisten Stimmen übernommen. 
+    Dieses Meeting findet im Anschluss eines Dailys oder BI-Weeklys statt, wenn seit dem Letzten neue Tickets hinzugefügt wurden.
+
+    In \glqq Dailys\grqq~wird von jedem Teammitglied der aktuelle Stand erläutert und Probleme, wie blockierende Tickets oder aufgetretene 
+    Unklarheiten angesprochen. Wenn nötig, kann daraufhin auch der Plan des aktuellen Sprints angepasst werden. Normalerweise wird dieses Meeting 
+    dem Namen entsprechend täglich abgehalten, ist jedoch in unserem Fall nur einmal wöchentlich im Rahmen der Vorlesung vorgesehen. Um einen 
+    häufigeren Austausch zu ermöglichen haben wir zusätzlich ein \glqq BI-Weekly\grqq~eingeführt, welches inhaltlich dem Daily gleicht und ebenso 
+    wöchentlich abgehalten wird.
+
+    Zum Abschluss eines Sprints finden \glqq Review\grqq~und danach \glqq Retro\grqq~statt. Im Review wird besprochen, was im Sprint erreicht 
+    wurde und wie dies den allgemeinen Fortschritt des Projektes beeinflusst. Dabei wird auch besprochen, was als Nächstes zu Erreichen ist. 
+    Als Resultat dessen kann der Produkt Backlog beeinflusst werden und Ideen für das Planning des nächsten Sprints gesammelt werden. 
+    In der \glqq Sprint-Retrospektive\grqq~oder kurz \glqq Retro\grqq~wird die Zusammenarbeit im Team bewertet. Dabei wird besprochen, was gut 
+    lief und weitergeführt werden sollte, welche Probleme aufgetreten sind, was im weiteren Verlauf des Projektes vermieden werden sollte sowie 
+    potenzielle Neuerungen, die eingeführt werden sollten. Dies resultiert in sogenannten Action Items, welche das besprochene konkretisieren.
```

Comment:
Auch noch erwähnen, dass in der retro verschiedene spielerische aufarbeitungen stattgefunden haben (diese aber nicht weiter benennen)